providers/unboundid/src/java/org/sakaiproject/unboundid/EntryContainerRdnToUserTypeMapper.java

```diff
@@ -79,7 +79,13 @@ public String mapLdapEntryToSakaiUserType(LDAPEntry ldapEntry,
 			DN containerDN = dn.getParent();
 			RDN[] containerRDNs = containerDN.getRDNs();
 			for (RDN rdn : containerRDNs) {
-				String mappedValue = mapRdn(rdn.toNormalizedString());
+				final String[] attrValues = rdn.getAttributeValues();
+				String mappedValue = null;
+				for (String av : attrValues) {
```

Comment:
I'd personally just stick rdn.getAttributeValues() in the loop statement and save a line.

Reply:
yup agree. thanks